packages/dashboard/src/portal/views/Transfer.vue

```diff
@@ -135,23 +228,72 @@ export default class TransferView extends Vue {
                 );
               } else if (section === "system" && method === "ExtrinsicFailed") {
                 this.$toasted.show("Transfer failed!");
-                this.loadingTransfer = false;
+                this.loadingTransferAddress = false;
               }
             });
           }
         }
       },
     ).catch(err => {
       this.$toasted.show(err.message);
-      this.loadingTransfer = false;
+      this.loadingTransferAddress = false;
     });
   }
 
+  async transferTFTWithTwinID() {
+    const twinDetails = await this.queryClient.twins.get({ id: parseInt(this.receptinTwinId) });
+    const injector = await web3FromAddress(this.$store.state.credentials.account.address);
```

Comment:
it's still using the extension while it's gone on the development branch, so please merge the development branch into this branch and use the mnemonic instead